app/_how-tos/enable-rbac-with-admin-api.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Enable RBAC with the Admin API
+content_type: how_to
+
+products:
+    - gateway
+tier: enterprise
+works_on:
+    - on-prem
+
+related_resources:
+  - text: Gateway RBAC entity
+    url: /gateway/entities/rbac/
+  - text: Gateway Group entity
+    url: /gateway/entities/group/
+  - text: Gateway Admin entity
+    url: /gateway/entities/admin/
+  - text: Workspaces entity
+    url: /gateway/entities/workspace
+
+
+tldr: 
+  q: How do I configure RBAC?
+  a: |
+     To configure RBAC, create a Super Admin user, then enable RBAC on {{site.base_gateway}} by setting the `enable_rbac` setting to `on`.
+
+min_version:
+    gateway: '3.4'
+prereqs:
+    inline:
+      - title: Configure environment variables
+        content: |
+            Set the `user_token`: 
+            * `USER_TOKEN`: The authentication token to be presented to the Admin API.
+            For example: 
+            ```sh
+            export USER_TOKEN=my-admin-token
+            ```
+---
+
+
+## 1. Create an RBAC Super Admin
+
+In {{site.base_gateway}} A Super Admin has the ability to manage Roles and permissions across Workspaces. Because the username `super-admin` matches the `super-admin` RBAC Role, the new user is automatically added to the `super-admin` Role. 
```

Comment:
```suggestion
In {{site.base_gateway}} A Super Admin has the ability to manage [Roles and permissions](/gateway/entities/rbac/#what-is-rbac) across Workspaces. Because the username `super-admin` matches the `super-admin` RBAC Role, the new user is automatically added to the `super-admin` Role. 
```

I know the RBAC page is already linked in the TL;DR and related links, but a user could skip all of those.

I'd like to know why you have to create the Super Admin user first. Is it because only a Super Admin user can enable RBAC?